### PR TITLE
bugfix: fix some failed test case

### PR DESCRIPTION
**Pull request by lynch1981**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Comment by lynch1981** [comment]
I tried fixing a few test cases @zhuizhuhaomeng

**Review by zhuizhuhaomeng — APPROVED** [review]

File: t/066-socket-receiveuntil.t [hunk 1]
```diff
@@ -126,7 +126,7 @@ GET /t
 qq{connected: 1
 request sent: 57
 read: HTTP/1.1 200 OK
-read: Server: nginx
+read: Server: openresty
```
Comment (zhuizhuhaomeng) [inline t/066-socket-receiveuntil.t:129]: This is not necessary.
The CI did apply the patch for the server name.

File: t/066-socket-receiveuntil.t [hunk 2]
```diff
@@ -196,7 +196,7 @@ GET /t
 qq{connected: 1
 request sent: 57
 read: HTTP/1.1 200 OK\r
-Server: nginx\r
+Server: openresty\r
```
Comment (zhuizhuhaomeng) [inline t/066-socket-receiveuntil.t:199]: ditto

File: t/138-balancer-upstream-bind.t [hunk 3]
```diff
@@ -135,7 +135,6 @@ ok
 
 --- request
     GET /t
---- response_body
-ok
+--- error_code: 500
```
Comment (zhuizhuhaomeng) [inline t/138-balancer-upstream-bind.t:138]: which version of nginx did you use?

**Review by zhuizhuhaomeng — CHANGES_REQUESTED** [review]

File: t/066-socket-receiveuntil.t (hunk 1 above)
Comment (lynch1981) [reply t/066-socket-receiveuntil.t]: got it

File: t/066-socket-receiveuntil.t (hunk 2 above)
Comment (lynch1981) [reply t/066-socket-receiveuntil.t]: got it

**Review by zhuizhuhaomeng — APPROVED** [review]

**Pull request merged**